### ISSUE TITLE
Topic level documentation for preview modes

### ISSUE THREAD

**Issue by kaedroho** [wagtail/wagtail#5561]
Wagtail has a feature where you can define multiple "preview" modes for a page. This is useful for previewing sub routes of a page defined with `RoutablePageMixin`.

Reference docs for the parts that need overriding can be found at: https://docs.wagtail.io/en/v2.6.1/reference/pages/model_reference.html#wagtail.core.models.Page.preview_modes

## Working on this

This would be a good task for someone with real-world experience with preview modes and-or routable pages. View our [contributing guidelines](https://docs.wagtail.org/en/latest/contributing/index.html), add a comment to the issue once you're ready to start.

**Comment by AdityaSharma-Git3207** [comment]
I checked this and it still seems relevant, Wagtail still supports page preview modes (documented in the model reference), but I couldn’t find topic-level docs showing how to use them in practice, especially with RoutablePageMixin.
I’m planning to open a docs PR adding a short guide + example if that sounds OK.

**Comment by thibaudcolas** [comment]
@AdityaSharma-Git3207 yes, this is still relevant. This would be best done by someone with real-world experience using preview modes / RoutablePageMixin, as how-to pages are best written with real use cases in mind.
